modules/api/pkg/ee/clusterbackup/storage-location/provider.go

```diff
@@ -107,7 +107,7 @@ func (p *BackupStorageProvider) Create(ctx context.Context, userInfo *provider.U
 		return nil, errors.New("a user is missing but required")
 	}
 	cbslFullName := fmt.Sprintf("%s-%s", cbslName, projectID)
-	secretName := fmt.Sprintf("credential-%s-", cbslFullName)
+	secretName := fmt.Sprintf("credential-%s", cbslFullName)
```

Comment:
Please explain why this change is required here and could this drift in naming pattern have any affect on existing clusters?